libraries/provision/ansible/playbooks/manage-testserver-java-desktop.yml

```diff
@@ -0,0 +1,14 @@
+- name: Manage TestServer Java Desktop as Daemon Service
+  hosts: testserver
+  vars:
+    service_status:
+    package_name:
+
+  tasks:
+  - debug: msg="JAVA_HOME location is {{ ansible_env.JAVA_HOME }}"
+  - debug: msg="JSVC_HOME location is {{ ansible_env.JSVC_HOME }}"
+
+  - name: Stop TestServer Java Daemon Service
+    shell: "~/javatestserver/daemon_manager.sh stop ~/javatestserver/{{ package_name }}.jar ~/javatestserver/output {{ ansible_env.JAVA_HOME }} {{ ansible_env.JSVC_HOME }}"
+    when: "{{ service_status | search('stop') }}"
+    ignore_errors: yes
```

Comment:
no, it's intentional setting the ignore_errors to true, because if daemon stopped by other reason or prior to this command, this command to stop a "stopped daemon service" will throw error, and we are ok with the error.